src/code/z_rcp.c

```diff
@@ -921,7 +921,7 @@ Gfx* func_800937E4(Gfx* gfx) {
     return gfx;
 }
 
-Gfx* func_80093808(Gfx* gfx) {
+Gfx* Gfx_SetupTitleCardDL(Gfx* gfx) {
     gSPDisplayList(gfx++, sSetupDL[0x34]);
     gDPSetColorDither(gfx++, G_CD_DISABLE);
```

Comment:
with these generic setup display lists it will be better (but very hard) to name them after what the preset actually means/what its achieving. In this case its only used by one thing, which is the title card dl, but that wont always necessarily be the case (in a modding context). Its designed to be a generic dl that can be used for many things

putting all of what these setup dls do into words is going to be very hard though, and why all of them are still unnamed right now

Reply:
Figured as much, but thought the same since it was used exclusively there.

Reply:
yeah it is. but i dont think its a display list that will work exclusively for title card textures. its just what the final game ended up using it for in this case

Reply:
another way to phrase my thoughts: if this were really exclusive to title cards it probably wouldnt be a function in z_rcp.c and would just be inlined in the title card itself

Reply:
Removed